content/docs/1.7.0/high-availability/rwx-volume-fast-failover.md

```diff
@@ -20,3 +20,25 @@ For more information, see https://github.com/longhorn/longhorn/issues/6205.
 > kubectl -n longhorn-system delete lease pvc-2ce4e82e-7ccc-46c0-90a8-a141501fbf93
 > ```
 > See, for example, https://github.com/longhorn/longhorn/issues/9093.
+
+### Performance Testing
+
+If your cluster has many RWX volumes, enabling the RWX Volume Fast Failover feature can have a significant impact on system performance, particularly in terms of `API requests`, `RPC rate`, `CPU`, and `memory` usage. Below are some key observations from testing in the environment:
```

Comment:
Hi, removed 'can' as it appears it does have a performance effect. If it might not, then those circumstances should be documented too. 

Removed 'some'. If there are other key observations they should be here as well.

Removed particularly. If there are other areas with high impact then they should be documented here.

```suggestion
If your cluster has many RWX volumes, enabling the RWX Volume Fast Failover feature has a high impact on system performance in terms of `API requests`, `RPC rate`, `CPU`, and `memory` usage. Below are key observations from testing:
```